tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationStringTest.cs

```diff
@@ -83,5 +85,76 @@ private string TranslateError (string culture, string errorCode)
 			PropertyInfo propertyInfo = typeof (MSBStrings).GetProperty (errorCode);
 			return (string) propertyInfo.GetValue (null, null);
 		}
+
+		private List<string> commonIgnoreList { get; set; } = null;
```

Comment:
We don't use `private` when it's the default:

```suggestion
		List<string> commonIgnoreList { get; set; } = null;
```

Reply:
Do we even need a property at all? Can be an instance variable.

Reply:
I made this one a property so we could update it the first time and then use the values in for the other tests